### PR TITLE
fix: add gurjeetsinghvirdee

### LINKED ISSUE #950: Gurjeet Singh Virdee
## Profile Screenshot

my-github-profile

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
resolves #950 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/GurjeetSinghVirdee.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/73753957/147848925-4655ce63-2950-4deb-9931-e8140909bf74.png",
+  "issueId": 950,
+  "name": "Gurjeet Singh Virdee",
+  "username": "gurjeetsinghvirdee"
+}
```
Comment (syedsohan) [inline profiles/GurjeetSinghVirdee.json:6]:
```suggestion
}

```

Add newline after line number `6`.

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — APPROVED** [review]

File: profiles/GurjeetSinghVirdee.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/73753957/147848925-4655ce63-2950-4deb-9931-e8140909bf74.png",
```
Comment (krishguptadev) [inline profiles/GurjeetSinghVirdee.json:2]: I prefer an image more like =>

image

```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147851903-7af25096-b607-42a2-bbfb-e7e77c53ecb6.png",
```

**Review by krishguptadev — APPROVED** [review]

File: profiles/GurjeetSinghVirdee.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91655303/147851903-7af25096-b607-42a2-bbfb-e7e77c53ecb6.png",
```
Comment (Panquesito7) [inline profiles/GurjeetSinghVirdee.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/73753957/147848925-4655ce63-2950-4deb-9931-e8140909bf74.png",
```

File: profiles/GurjeetSinghVirdee.json (hunk 2 above)
Comment (Panquesito7) [inline profiles/GurjeetSinghVirdee.json:2]: Sorry if I reverted this, Krish, didn't see this properly. 😅
```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147851903-7af25096-b607-42a2-bbfb-e7e77c53ecb6.png",
```

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**